registry/registry_test.go

```diff
@@ -98,3 +121,221 @@ func TestGracefulShutdown(t *testing.T) {
 		t.Error("Body is not {}; ", string(body))
 	}
 }
+
+func TestGetCipherSuite(t *testing.T) {
+	resp, err := getCipherSuites([]string{"TLS_RSA_WITH_AES_128_CBC_SHA"})
+	if err != nil || len(resp) != 1 || resp[0] != tls.TLS_RSA_WITH_AES_128_CBC_SHA {
+		t.Error("did not return expected cipher suite constant")
+	}
+
+	resp, err = getCipherSuites([]string{"TLS_RSA_WITH_AES_128_CBC_SHA", "TLS_AES_128_GCM_SHA256"})
+	if err != nil || len(resp) != 2 ||
+		resp[0] != tls.TLS_RSA_WITH_AES_128_CBC_SHA || resp[1] != tls.TLS_AES_128_GCM_SHA256 {
+		t.Error("did not return expected cipher suite constant")
+	}
```

Comment:
Can we make this a bit more helpful for folks to understand why the test might be failing in the future? Say, by printing what _is expected_ to be returned or at least some sort of "hint" like "expected 1 cipher" or something like that? We have two similar test cases here which both might return an error with the same error content